includes/settings/settings-ppec.php

```diff
@@ -376,4 +376,11 @@
 			'drop' => __( 'Do not send line items to PayPal', 'woocommerce-gateway-paypal-express-checkout' ),
 		),
 	),
+	'checkout_product' => array(
```

Comment:
can we move this under **Display Settings** section instead of **Advanced Settings** ?